flash_examples/face_detection.py

```diff
@@ -0,0 +1,50 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import flash
+from flash.core.data.data_module import DataModule
+from flash.core.utilities.imports import _FASTFACE_AVAILABLE
+from flash.image import FaceDetector
+from flash.image.face_detection.data import FaceDetectionPreprocess
+
+if _FASTFACE_AVAILABLE:
+    import fastface as ff
+else:
+    raise ModuleNotFoundError("Please, pip install -e '.[image]'")
+
+# 1. Create the DataModule
+train_dataset = ff.dataset.FDDBDataset(source_dir="data/", phase="train")
+val_dataset = ff.dataset.FDDBDataset(source_dir="data/", phase="val")
+
+datamodule = DataModule.from_data_source(
```

Comment:
Once a `DefaultDataSources.DATASET` is added and we have a `FaceDetectionData`, this can just become:
```
FaceDetectionData.from_datasets(...)
```
No need to pass the preprocess or "fastface"